guide/source/3.0-migration.md

```diff
@@ -41,3 +41,29 @@ The best way to follow the progress is by checking the "[What's left until an of
 ### When will Meteor 3.0 be ready?
 
 Our plan is to release the beta version by the end of Q4 2023. An official version will depend a lot on user feedback, but we aim to release it by the end of Q1 2024.
+
+
+### How do I migrate my package to be compatible with Meteor 3.0?
+
+For the packages that are client only
+or that are do not using Meteor packages that will become async
+or are already using `async` & `await` pattern.
+
+The migration will look like this:
+
+```js
+// in you package.js
+Package.onUse((api) => {
+  api.versionsFrom(['1.10', '2.3', '3.0-alpha.19']);
+  //                               ^^^^^^^ for testing your package with meteor 3.0
+
+  api.versionsFrom(['1.10', '2.3', '3.0']);
+  //                              ^^^^^^^ for meteor 3.0
+
+```
+Then you can publish your package and test it with Meteor 3.0.
+
+If in your package you are using Meteor packages that will become async,
+you will need to migrate your package to use `async` & `await` pattern.
+
+You can follow a more in depth guide on how to migrate your package to be compatible with Meteor 3.0 [here](/prepare-meteor-3.0#Changes-for-packages).
```

Comment:
Could you add some examples of migrated packages? For instance, `quave:migrations` and `percolate:synced-cron` that it is in progress.

Reply:
Sure!